src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/ChangePermissionsController.cs

```diff
@@ -0,0 +1,77 @@
+﻿using FluentValidation;
+using FluentValidation.AspNetCore;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Provider.PR.Web.Authorization;
+using SFA.DAS.Provider.PR.Web.Infrastructure;
+using SFA.DAS.Provider.PR.Web.Infrastructure.Services;
+using SFA.DAS.Provider.PR.Web.Models.AddEmployer;
+using SFA.DAS.Provider.PR.Web.Models.Session;
+
+namespace SFA.DAS.Provider.PR.Web.Controllers.AddEmployer;
+
+[Authorize(Policy = nameof(PolicyNames.HasContributorOrAbovePermission))]
+
+[Route("/{ukprn}/addEmployer/changePermissions", Name = RouteNames.ChangePermissions)]
```

Comment:
I think we should use AddPermissions here as ChangePermissions will have completely different meaning.